src/main/java/io/specto/hoverfly/junit/dsl/RequestedServiceBuilder.java

```diff
@@ -0,0 +1,21 @@
+package io.specto.hoverfly.junit.dsl;
+
+import io.specto.hoverfly.junit.dsl.matchers.PlainTextFieldMatcher;
+
+public class RequestedServiceBuilder extends AbstractServiceBuilder {
```

Comment:
Why do we need a new class for this? The matcher schema should be exactly the same between verify and the simulation, so you should be able to use the ordinary DSL classes / builders?